index.js

```diff
@@ -150,7 +155,7 @@ function filter(patterns, opts) {
 
     fp = utils.unixify(fp, opts);
     while (i < len) {
-      var fn = matcher(patterns[i++], opts);
+      var fn = patterns[i++];
       if (!fn(fp)) {
         res = false;
```

Comment:
> why not bump patch instead of minor?

of course lol, I was pretty tired when I wrote that. seems I was looking for the most complicated way to do things possible lol